test/commands/user/authenticate_from_omniauth_test.rb

```diff
@@ -101,4 +101,56 @@ class User::AuthenticateFromOmniauthTest < ActiveSupport::TestCase
     assert user.persisted?
     refute_equal "user22", user.handle
   end
+
+  test "recalculate pull request reputation for uid matches that change the github_username" do
+    user = create :user, provider: "github", uid: "111", github_username: nil
+    auth = stub(provider: "github", uid: "111", info: stub(nickname: "user22"))
+
+    assert_enqueued_with(job: AwardPullRequestReputationJob, args: [user], queue: 'reputation') do
```

Comment:
@iHiD Could you check if I've the tests I've added regarding the queueing of the job work like I intend to? I made a mistake with them in a previous PR.